cmd/containerboot/main.go

```diff
@@ -651,6 +651,8 @@ func tailscaleUp(ctx context.Context, cfg *settings) error {
 	}
 	if cfg.Routes != "" {
```

Comment:
Could also convert this into a pointer and have the three states. Unset, set and empty.

Reply:
I was thinking about sth like that, but then that would be inconsistent with the `acceptDNS` for example where, if it's unset we treat it the same as explicitly set to false. Or is that not a good comparison

Reply:
just thinking about the backwards compat problem, not sure if its applicable here

Reply:
I can't think of a standard flow where this would be set once and not again. Maybe there is some mystery reason for someone to, instead of providing the value via env var, to ssh to the container at a specific point in time and expose a route that way, not sure why anyone would do that, but we'd probably break that

Reply:
yeah, I guess there isn't a particular reason to unset the subnet routes if the env var is not provided because we again assume that the standard flow would have all config opts. So maybe it's more valuable to have the 3 values for backwards compat

Reply:
should be better now